content/en/real_user_monitoring/browser/modifying_data_and_context.md

```diff
@@ -74,15 +74,17 @@ window.DD_RUM &&
 {{% /tab %}}
 {{< /tabs >}}
 
-2. You must start views for each new page or route change (for single-page applications). RUM data is collected when the view starts. Optionally, define the associated view name, service name, and version.
+2. You must start views for each new page or route change (for single-page applications). RUM data is collected when the view starts. Starting from [version 4.13.0](17), you can also optionally define the associated service name, and version.
 
    - View Name: Defaults to the page URL path.
    - Service: Defaults to the default service specified when creating your RUM application.
    - Version: Defaults to the default version specified when creating your RUM application.
 
    For more information, see [Setup Browser Monitoring][4].
 
-The following example manually tracks the page views on the `checkout` page in a RUM application. Use `checkout` for the view name and associate the `purchase` service with version `1.2.3`.
+<details open>
+  <summary>Latest version</summary>
+The following example manually tracks the page views on the <code>checkout</code> page in a RUM application. Use <code>checkout</code> for the view name and associate the <code>purchase</code> service with version <code>1.2.3</code>.
```

Comment:
Sorry, this might be a silly question, but is version 1.2.3 an example value or is it the actual version number of the purchase service.

Reply:
Both the `purchase` service and version `1.2.3` are examples

Reply:
Got it. I added a suggestion for a potential edit.